File: aula5/Em_java/Chefe.java
public class Chefe extends Personagem{
    public Chefe(String nome){
        super(nome, 200, 99, 20);
    }
    @Override
    public String habilidade(){
        return "Bafo de fogo";
    }
    @Override
    public void ficha(){
        System.out.println("Chefe");
        super.ficha();
    }
}

File: aula5/Em_python/Chefe.py
from Personagem import Personagem
class Chefe(Personagem):
    def __init__(self, nome: str):
        super().__init__(nome, 200, 32, 20)
    def habilidade(self) -> str:
        return "golpe_de_furia"
    def ficha(self):
        print("Ficha do Chefe:")
        super().ficha()